### DISCUSSION TITLE
Emerald's spawn

### DISCUSSION THREAD

**Discussion by svesnav** [DerFrZocker/Ore-Control#16] [Q&A] [answered]
Hi, how to make emeralds spawn in all biomes? I can't find solution!

**Comment by DerFrZocker** [comment, marked as answer]
This cannot be done directly with Ore-control, you can do this by using my other Plugin Custom-ore-generator  to generated them, or re-configurated an other ore which you don't want with Ore-control to generated emeralds instead.

**Reply by svesnav** [reply]
I am using Paper 1.20.1, but Custom-ore-generator doesn't supported it. So how to re-configurated an other ore to generate emeralds instead? This mean, that other ore will not spawn, if were spawn as emerald ore?
